test/e2e/specs/site-editor/styles.spec.js

```diff
@@ -54,6 +54,6 @@ test.describe( 'Styles', () => {
 		await paddingControl.fill( '2' );
 
 		// Check the padding value
-		await expect( block ).toHaveCSS( 'padding-left', '35.4644px' );
+		await expect( block ).toHaveCSS( 'padding-left', '35.4932px' );
```

Comment:
That's true, but I don't have much time left today, so I'd appreciate it if you could just merge it as is, or update the tests 🙏

Reply:
I think it could be achieved with something like this, although I am not sure if those are the best selectors:

```js
// find the second padding control
await page.getByRole("button", { name: "Set custom size" }).nth(1).click();
await page.locator('[id*="inspector-input-control"]').fill("35");

// Check the padding value
await expect(block).toHaveCSS("padding-left", "35px");
```